Added Dual Engine API Ref (IE Mode) (#2859)
MicrosoftDocs/edge-developer: microsoft-edge/dualengine/index.md

Reviewer: i might push a comment to change this filename from index.md to something specific/descriptive like intro.md per
https://review.learn.microsoft.com/en-us/help/platform/seo-optimize-urls?branch=main#name-your-files

Reviewer: done; renamed the non-Ref index.md as intro.md

---
description: WebView2 Win32 C++ Reference
title: WebView2 Win32 C++ Reference
ms.date: 09/07/2023
keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edge, ICoreWebView2, ICoreWebView2Controller, browser control, edge html
---

# Reference (WebView2 Win32 C++)

> The Microsoft Edge WebView2 control enables you to host web content in your application using [Microsoft Edge \(Chromium\)](https://www.microsoftedgeinsider.com) as the rendering engine.  For more information, see [Overview of Microsoft Edge WebView2](/microsoft-edge/webview2/index)) and [Getting Started with WebView2](/microsoft-edge/webview2/gettingstarted/win32).  [ICoreWebView2](icorewebview2.md) is a great place to start learning the details of the API.

Reviewer: ```suggestion
The Microsoft Edge WebView2 control enables you to host web content in your application using [Microsoft Edge \(Chromium\)](https://www.microsoftedgeinsider.com) as the rendering engine.  For more information, see [Overview of Microsoft Edge WebView2](/microsoft-edge/webview2/index) and [Getting Started with WebView2](/microsoft-edge/webview2/gettingstarted/win32).  [ICoreWebView2](icorewebview2.md) is a great place to start learning the details of the API.
```
* extra )

Reviewer: no extra )) in /dualengine/ dir

---
title: Introduction to Microsoft DualEngine
description: Host web content in your Win32 apps with the Microsoft DualEngine interface.
author: MSEdgeTeam
ms.author: edgededev
ms.topic: conceptual
ms.prod: microsoft-edge
ms.localizationpriority: high
ms.technology: windows-integration
ms.date: 11/06/2023
---
# Introduction to Microsoft DualEngine API

> The Microsoft DualEngine API allows you to embed and control a Internet Explorer within your app.

Reviewer: ```suggestion
The Microsoft DualEngine API allows you to embed and control an Internet Explorer instance within your app.
```
alts:
* The Microsoft DualEngine API allows you to embed and control an Internet Explorer instance within your app.
* The Microsoft DualEngine API allows you to embed and control an Internet Explorer control within your app.
* The Microsoft DualEngine API allows you to embed and control an Internet Explorer window within your app.
* The Microsoft DualEngine API allows you to embed and control an Internet Explorer pane within your app.

Reviewer: update source file too?  Updated to:
The Microsoft DualEngine API enables you to host and control an Internet Explorer instance within your app.

i changed to that wording in the gen'd .md file today, you might want to apply that change to the actual source file @AndrewLakeMSFT

Reviewer: The .md is a static, not gen'd file.  No longer "index.md"; now "intro.md".  

Latest Files Changed tab includes the word 'instance':

Use the Microsoft DualEngine API to embed and control an Internet Explorer instance within your app.  The DualEngine API provides a browsing experience that's equivalent to a normal Internet Explorer tab, to provide increased compatibility with legacy websites.

The DualEngine API differs from similar APIs like the Web Browser Control, and requires distinct development work.

> The DualEngine API differs from similar APIs like the Web Browser Control by providing a browsing experience

Reviewer: ```suggestion
The DualEngine API differs from similar APIs like the Web Browser Control by providing a browsing experience that's 
```

Reviewer: The DualEngine API provides a browsing experience that's equivalent to a normal Internet Explorer tab, to provide increased compatibility with legacy websites.

The DualEngine API differs from similar APIs like the Web Browser Control, and requires distinct development work.

> equivalent to a normal Internet Explorer tab for increased compatibility with legacy websites. However this

Reviewer: ```suggestion
equivalent to a normal Internet Explorer tab for increased compatibility with legacy websites. However, this
```

Reviewer: equivalent to a normal Internet Explorer tab, to provide increased compatibility with legacy websites.

The DualEngine API differs from similar APIs like the Web Browser Control, and requires distinct development work.

increase in compatibility comes at the cost of increased developer overhead.

> To start building with the DualEngine APi, see [Getting Started with DualEngine API](get-started.md)

Reviewer: ```suggestion
To start building with the DualEngine APi, see [Getting Started with DualEngine API](get-started.md).
```

Reviewer: See also:
* [Getting started with the DualEngine API](./get-started.md)
